src/sentry/uptime/endpoints/uptime_ips.py

```diff
@@ -0,0 +1,24 @@
+from django.http.response import HttpResponse
+from rest_framework.request import Request
+
+from sentry.api.api_publish_status import ApiPublishStatus
+from sentry.api.base import Endpoint
+
+UPTIME_IP_ADDRESSES = [
+    "34.169.179.115",
+    "35.237.134.233",
+    "34.85.249.57",
+    "34.107.93.3",
+    "35.204.169.245",
+]
+
+
+class UptimeIpsEndpoint(Endpoint):
+    publish_status = {
+        "GET": ApiPublishStatus.PRIVATE,
```

Comment:
would be nice to make public, but it doesn't have a schema since it's not normal JSON